### PR TITLE
2460 DynamoPSy split & rename

### LINKED ISSUE #2460: Splitting DynamoPSy from `dynamo0p3.py`
sub-issue of #2235

### PR DESCRIPTION
**Pull request by oakleybrunt**
Closes #2460

### CODE REVIEW

File: examples/lfric/eg2/loop_fuse_trans.py [hunk 1]
```diff
@@ -50,10 +50,10 @@ def trans(psy):
 
     :param psy: the PSy object that PSyclone has constructed for the \
```
Comment (mo-lottieturner) [inline examples/lfric/eg2/loop_fuse_trans.py:51]: remove backslash

File: examples/lfric/eg2/module_inline_trans.py [hunk 2]
```diff
@@ -48,10 +48,10 @@ def trans(psy):
 
     :param psy: the PSy object that PSyclone has constructed for the \
```
Comment (mo-lottieturner) [inline examples/lfric/eg2/module_inline_trans.py:49]: remove backslash

File: examples/lfric/eg2/print_psyir_trans.py [hunk 3]
```diff
@@ -46,10 +46,10 @@ def trans(psy):
 
     :param psy: the PSy object that PSyclone has constructed for the \
```
Comment (mo-lottieturner) [inline examples/lfric/eg2/print_psyir_trans.py:47]: remove backslash

File: src/psyclone/tests/domain/lfric/lfric_psy_test.py [hunk 4]
```diff
@@ -53,7 +52,7 @@
 
 class DummyInvokeInfo():
     '''A dummy class that contains the minimum information required to
-    allow DynamoPSy to be called.
+    allow LFRicPSy to be called.
 
     :param str name: the name to use for this dummy invoke. Defaults \
```
Comment (mo-lottieturner) [inline src/psyclone/tests/domain/lfric/lfric_psy_test.py:57]: remove backslash

**Review by mo-lottieturner — CHANGES_REQUESTED** [review]
Mostly backslashes needing removing

**Comment by arporter** [comment]
This branch is getting horribly out of date (and thus hard to merge). Are you still planning to complete this @oakleybrunt and @mo-lottieturner?

**Review by mo-lottieturner — APPROVED** [review]

**Comment by mo-lottieturner** [comment]
Approved, but I'm not authorised to merge this onto trunk

**Comment by mo-lottieturner** [comment]
> Thanks both. I've just made @mo-lottieturner a 'maintainer' - does that then permit you to merge this PR? If not, I'll upgrade you to 'admin' :-)

Yep, looks like it! How do I know that the integration testing has passed? Looks like Oakley has updated the changelog

**Comment by arporter** [comment]
You go here: https://github.com/stfc/PSyclone-mirror/actions I've just given you read access (you might have an email invitation?).

**Comment by arporter** [comment]
You'll see that everything apart from NEMOv5 was OK and NEMOv5 is expected to fail currently so that's OK.

**Comment by arporter** [comment]
(You also need to build and update the User Guide in the root directory - see https://github.com/stfc/PSyclone/wiki/CodeReview#merging-a-branch-to-master)

**Comment by mo-lottieturner** [comment]
Merged and branch deleted

**Pull request merged**